src/parsers/manifest/dash/node_parsers/Period.ts

```diff
@@ -50,14 +52,79 @@ export interface IPeriodAttributes {
   xlinkActuate? : string;
 }
 
+/**
+ * Parse the EventStream node to extract Event nodes and their
+ * content.
+ * @param {Element} element
+ */
+function parseEventStream(element: Element): IStreamEvent[] {
+  const streamEvents: IStreamEvent[] = [];
+  const attributes: { schemeId?: string;
+                      timescale: number;
+                      value?: string; } =
+                      { timescale: 1 };
+
+  for (let i = 0; i < element.attributes.length; i++) {
+    const attribute = element.attributes[i];
+    switch (attribute.name) {
+      case "schemeIdUri":
+        attributes.schemeId = attribute.value;
+        break;
+      case "timescale":
+        attributes.timescale = parseInt(attribute.value, 10);
```

Comment:
It makes me think that I added in another PR the `parseMPDInteger` function, which also returns a warning when the attribute is not in an integer form. We might want to use it there also.